src/wasm-s-parser.h

```diff
@@ -1470,6 +1469,12 @@ class SExpressionWasmBuilder {
       } else if (inner[0]->str() == IMPORT) {
         importModule = inner[1]->str();
         importBase = inner[2]->str();
+        auto im = make_unique<Import>();
+        im->kind = ExternalKind::Memory;
+        im->module = importModule;
+        im->base = importBase;
+        im->name = importModule; // + importBase;
```

Comment:
OK, I took it as a reminder for later to actually construct a string concatening importModule.importBase. Will remove both.